Lib/typing.py

```diff
@@ -250,6 +250,9 @@ def _collect_parameters(args):
     """
     parameters = []
     for t in args:
+        # A bare Python class isn't generic.
```

Comment:
I am not sure whether this comment is useful or misleading. Yes, Python types are not generic, but the true reason of adding this check is that the type has the same attribute as instance if they are defined as descriptors in a type. There are no any comments to other uses of `isinstance(t, type)` or `PyType_Check(t)`, so I am wondering why this case is special.

Reply:
OK, I'll update the comment instead to say ``We don't want __parameters__ descriptor of a bare Python class.``.